xtask/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "xtask"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.75"
```

Comment:
Can we use `thiserror` instead of `anyhow`? In this way we can set our own kinds of errors

Reply:
I would strongly recommend against using `thiserror` for a binary crate, as there is not much reason here to explicitly define error variants for programmatic scrutiny.
It's more important that the errors get propagated up to users with appropriate context, so they understand fully what caused the error and what we were trying to do when the error occurred.

`thiserror` is more appropriate for use in a library crate, where the consumer may want to handle the errors and/or return them up the stack.

Reply:
Yes, seems legit. [Here](https://github.com/dtolnay/anyhow#comparison-to-thiserror) an official reference of your comments